src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Symfony\Bridge\PhpUnit;
 
+use Symfony\Component\Config\Resource\ComposerResource;
```

Comment:
you can't use this class: it's not a dep of the bridge (and we don't want to add it as a dep to the bridge)
use copy/paste instead :)